SCPDiscordPlugin/Languages/ukrainian.yml

```diff
@@ -20,28 +20,28 @@ global_regex: [
   #"SCPs":"",
   "FoundationForces":"МОГ",
   "ChaosInsurgency":"Повстанець хаосу",
-  #"Scientists":"",
+  "Scientists":"Вчений",
   "ClassD":"Д-клас",
   #"Dead":"",
   "OtherAlive":"Інша команда",
 
   # Roles
-  #"None":"",
+  "None":"Неназвана роль",
   "Scp173":"SCP-173",
-  #"ClassD":"",
-  #"Spectator":"",
+  "ClassD":"Клас-Д",
+  "Spectator":"Глядач",
   "Scp106":"SCP-106",
   "NtfSpecialist":"МОГ Спеціаліст",
   "Scp049":"SCP-049",
-  #"Scientist":"",
+  "Scientist":"Науковець",
   "Scp079":"SCP-079",
   "ChaosConscript":"Новобранець Хаосу",
   "Scp096":"SCP-096",
   "SCP-0492":"SCP-049-2",
   "NtfSergeant":"Сержант МОГ",
   "NtfCaptain":"Капітан МОГ",
   "NtfPrivate":"Рядовий МОГ",
-  #"Tutorial":"",
+  "Tutorial":"Навчання",
```

Comment:
The ones commented out here shouldn't be used, they are just there for reference. If you look higher up you can see there are already translations for these phrases.

Reply:
oh, sorry